packages/devtools_app/lib/src/shared/vm_flags.dart

```diff
@@ -21,24 +21,23 @@ const profiler = 'profiler';
 const profilePeriod = 'profile_period';
 
 class VmFlagManager extends Disposer {
-  VmServiceWrapper get service => _service;
-  VmServiceWrapper _service;
+  VmServiceWrapper? get service => _service;
+  VmServiceWrapper? _service;
 
   ValueListenable get flags => _flags;
-  final _flags = ValueNotifier<FlagList>(null);
+  final _flags = ValueNotifier<FlagList?>(null);
 
-  final _flagNotifiers = <String, ValueNotifier<Flag>>{};
+  final _flagNotifiers = <String?, ValueNotifier<Flag>>{};
 
-  ValueNotifier<Flag> flag(String name) {
+  ValueNotifier<Flag>? flag(String name) {
     return _flagNotifiers.containsKey(name) ? _flagNotifiers[name] : null;
   }
 
   Future<void> _initFlags() async {
-    final flagList = await service.getFlagList();
+    final FlagList flagList = await service!.getFlagList();
     _flags.value = flagList;
-    if (flagList == null) return;
 
-    for (var flag in flagList.flags) {
+    for (var flag in flagList.flags!) {
```

Comment:
instead of force unwrapping, let's make this more null-safe by writing `(flagList.flags ?? <Flag>[])`

Reply:
done